### DISCUSSION TITLE
如何将主应用封装的自定义组件暴露给子应用去使用呢？

### DISCUSSION THREAD

**Discussion by stl1230** [jd-opensource/micro-app#401] [Q&A]
有大佬解答一下吗？插件系统没看懂，是能实现这样的需求

**Comment by bailicangdu** [comment]
通过数据通讯传递给子应用，所谓组件本质是一个函数

**Reply by stl1230** [reply]
那我在主应用main.js注册的组件，通过什么写法，可以让子应用去使用呢（vue的）

**Reply by bailicangdu** [reply]
通过数据通信传递到子应用，子应用通过vue.use注册组件  https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/data

**Reply by stl1230** [reply]
感谢大佬，可以了
